pkgs/sops-install-secrets/darwin.go

```diff
@@ -53,7 +53,7 @@ func SecureSymlinkChown(symlinkToCheck string, expectedTarget string, owner, gro
 // mydev=`hdiutil attach -nomount ram://$NUMSECTORS`
 // newfs_hfs $mydev
 // mount -t hfs $mydev /tmp/mymount
-func MountSecretFs(mountpoint string, keysGid int, userMode bool) error {
+func MountSecretFs(mountpoint string, keysGid int, useTmpfs bool, userMode bool) error {
```

Comment:
```suggestion
func MountSecretFs(mountpoint string, keysGid int, _useTmpfs bool, userMode bool) error {
```
maybe add underscore to indicate it is not used